http/rest-client-reactive/src/test/java/io/quarkus/ts/http/restclient/reactive/ReactiveRestClientProxyIT.java

```diff
@@ -19,7 +19,9 @@ public class ReactiveRestClientProxyIT {
 
     @Container(image = "${nginx.image}", port = 8090, expectedLog = "Configuration complete; ready for start up")
     static RestService proxy = new RestService()
-            .withProperty("_whatever", "resource_with_destination::/etc/nginx/|nginx.conf");
+            .withProperty("FORWARD_PROXY_RESOLVER", System.getProperty("forward-proxy-resolver"))
```

Comment:
this is fine, although I'd somehow expect `.withProperty("FORWARD_PROXY_RESOLVER", "${forward-proxy-resolver}")` to work as well